flutter/lib/ui/geometry.dart

```diff
@@ -836,40 +1079,82 @@ class RRect {
     return w < h ? w : h;
   }
 
-  /// The point halfway between the left and right and the top and bottom edges of this rectangle.
+  /// The point halfway between the left and right and the top and bottom edges
+  /// of this rectangle.
   Point get center => new Point(left + width / 2.0, top + height / 2.0);
 
+  double _get_min(double min, double radius1, double radius2, limit) {
+    double sum = radius1 + radius2;
+    if (sum > limit && sum != 0.0) {
+      return math.min(min, limit / sum);
+    }
+    return min;
+  }
+
+  void _scaleRadii() {
```

Comment:
This would be a good place for a // comment that explained what's going on here.